common/djangoapps/student/views.py

```diff
@@ -145,7 +145,7 @@ def csrf_token(context):
     if token == 'NOTPROVIDED':
         return ''
     return (u'<div style="display:none"><input type="hidden"'
-            ' name="csrfmiddlewaretoken" value="%s" /></div>' % (token))
+            ' name="csrfmiddlewaretoken" value="%s" /></div>' % token)
```

Comment:
Please undo this change. The file is unrelated and we shouldn't introduce noise.